src/rdf4cpp/rdf/parser/ParsingFlags.hpp

```diff
@@ -9,6 +9,10 @@ namespace rdf4cpp::rdf::parser {
 enum struct ParsingFlag : uint8_t {
     Strict = 1 << 0,
     NoParsePrefix = 1 << 1,
+    Turtle = 1 << 2,
```

Comment:
The flags could be encoded into the 2 bits after the `NoParsePrefix` bit. Behaviour would be essentially identical to how it is right now except that it saves some space (maybe relevant, idk yet) and it makes more sense semantically. Just encode turtle as `0b00` which automatically makes it the default value.